src/components/withWindowDimensions.js

```diff
@@ -11,6 +12,8 @@ const windowDimensionsPropTypes = {
 
         // Height of the window
         height: PropTypes.number.isRequired,
+
+        isSmallScreenWidth: PropTypes.bool,
```

Comment:
This should be outside of the `windowDimensions` prop. Like this:

```javascript
const windowDimensionsPropTypes = {
    // via withWindowDimensions
    windowDimensions: PropTypes.shape({
        // Width of the window
        width: PropTypes.number.isRequired,

        // Height of the window
        height: PropTypes.number.isRequired,
    }).isRequired,
    
    // Is the window width narrow, like on a mobile device?
    isSmallScreenWidth: PropTypes.bool.isRequired,
};
```

Also note that `isSmallScreenWidth` should be required. Our style guide mandates that any optional props should always have a default. In this case, the prop will always be provided by this HOC, so it's better to just make it a required prop.